unit-tests/py/rspy/test.py

```diff
@@ -344,8 +345,9 @@ def check_throws( _lambda, expected_type, expected_msg = None, abort_if_failed =
     try:
         _lambda()
     except Exception as e:
-        check_exception( e, expected_type, expected_msg, abort_if_failed )
-        return check_passed()
+        return check_exception( e, expected_type, expected_msg, abort_if_failed )
```

Comment:
I would expect `chec_exception` to fail the test instead of `print_stack`

Reply:
It does - but it still called `check_passed` before